.buildkite/pipeline.yaml

```diff
@@ -125,35 +53,13 @@ steps:
     secrets:
       INTEGRATION_TEST_BUILDKITE_TOKEN: integration_test_buildkite_api_token
       BUILDKITE_ANALYTICS_TOKEN: test_engine_suite_token
+    image: golang:latest
+    command: .buildkite/steps/tests.sh
     plugins:
       - kubernetes:
-          podSpec:
+          podTemplate: go-with-cache
+          podSpecPatch:
             serviceAccountName: integration-tests
-            volumes:
-              - name: agent-stack-k8s-config
-                configMap:
-                  name: agent-stack-k8s-config
```

Comment:
There is no need for our integration test to access our production controller config.